ts/store/reducers/wallet/outcomeCode.ts

```diff
@@ -0,0 +1,137 @@
+  outcomeCode: pot.none
+};
+
+// TODO: As refinement make the outcomeCodes list remote, like backend status.
+// This data structure replicates the future remote data structure.
+const OutcomeCodesPrintable = (): OutcomeCodes => ({
+  "0": {
+    status: "success"
+  },
+  "1": {
+    title: {
+      "en-EN": I18n.t("wallet.outcomeMessage.code1.title"),
+      "it-IT": I18n.t("wallet.outcomeMessage.code1.title")
+    },
+    description: {
+      "en-EN": I18n.t("wallet.outcomeMessage.code1.description"),
+      "it-IT": I18n.t("wallet.outcomeMessage.code1.description")
+    },
+    icon: genericError,
+    status: "errorBlocking"
+  },
+  "2": {
+    title: {
+      "en-EN": I18n.t("wallet.outcomeMessage.code2.title"),
+      "it-IT": I18n.t("wallet.outcomeMessage.code2.title")
+    },
+    description: {
+      "en-EN": I18n.t("wallet.outcomeMessage.code2.description"),
+      "it-IT": I18n.t("wallet.outcomeMessage.code2.description")
+    },
+    icon: authorizationDenied,
+    status: "errorBlocking"
+  },
+  "4": {
+    title: {
+      "en-EN": I18n.t("wallet.outcomeMessage.code4.title"),
+      "it-IT": I18n.t("wallet.outcomeMessage.code4.title")
+    },
+    description: {
+      "en-EN": I18n.t("wallet.outcomeMessage.code4.description"),
+      "it-IT": I18n.t("wallet.outcomeMessage.code4.description")
+    },
+    icon: sessionExpired,
+    status: "errorBlocking"
+  },
+  "7": {
+    title: {
+      "en-EN": I18n.t("wallet.outcomeMessage.code7.title"),
+      "it-IT": I18n.t("wallet.outcomeMessage.code7.title")
+    },
+    description: {
+      "en-EN": I18n.t("wallet.outcomeMessage.code7.description"),
+      "it-IT": I18n.t("wallet.outcomeMessage.code7.description")
+    },
+    icon: cardProblemOrOperationCanceled,
+    status: "errorBlocking"
+  },
+  "8": {
+    title: {
+      "en-EN": I18n.t("wallet.outcomeMessage.code8.title"),
+      "it-IT": I18n.t("wallet.outcomeMessage.code8.title")
+    },
+    icon: cardProblemOrOperationCanceled,
+    status: "errorBlocking"
+  },
+  "10": {
+    title: {
+      "en-EN": I18n.t("wallet.outcomeMessage.code10.title"),
+      "it-IT": I18n.t("wallet.outcomeMessage.code10.title")
+    },
+    description: {
+      "en-EN": I18n.t("wallet.outcomeMessage.code10.description"),
+      "it-IT": I18n.t("wallet.outcomeMessage.code10.description")
+    },
+    icon: authorizationDenied,
+    status: "errorBlocking"
+  }
+});
+
+// This fallback is used both for unexpected code and for code that we don't want to map specifically.
+const fallbackOutcomeCodes = (): OutcomeCode => ({
+  title: {
+    "en-EN": I18n.t("wallet.outcomeMessage.fallback.title"),
+    "it-IT": I18n.t("wallet.outcomeMessage.fallback.title")
+  },
+  description: {
+    "en-EN": I18n.t("wallet.outcomeMessage.fallback.description"),
+    "it-IT": I18n.t("wallet.outcomeMessage.fallback.description")
+  },
+  icon: genericError,
+  status: "errorBlocking"
+});
+
+const extractOutcomeCode = (
```

Comment:
Could you add a comment describing what this function does?

Reply:
Sure, added in [1f83b91](https://github.com/pagopa/io-app/pull/2842/commits/1f83b91c3a18d42d649a20dcc4e8a2703bb8bd0a)